src/main/java/org/dependencytrack/event/kafka/KafkaEventDispatcher.java

```diff
@@ -60,9 +60,9 @@ public Future<RecordMetadata> dispatchAsync(final Event event, final Callback ca
         } else if (event instanceof final ComponentRepositoryMetaAnalysisEvent e) {
             return dispatchAsyncInternal(KafkaEventConverter.convert(e), callback);
         } else if (event instanceof final OsvMirrorEvent e) {
-            return dispatchAsyncInternal(new KafkaEvent<>(KafkaTopics.MIRROR_OSV, e.ecosystem(), "", null), callback);
+            return dispatchAsyncInternal(new KafkaEvent<>(KafkaTopics.VULNERABILITY_MIRROR_COMMAND, e.ecosystem(), "", null), callback);
         } else if (event instanceof NistMirrorEvent) {
-            return dispatchAsyncInternal(new KafkaEvent<>(KafkaTopics.MIRROR_NVD, UUID.randomUUID().toString(), "", null), callback);
+            return dispatchAsyncInternal(new KafkaEvent<>(KafkaTopics.VULNERABILITY_MIRROR_COMMAND, UUID.randomUUID().toString(), "", null), callback);
```

Comment:
The event key will need to be the data source name, and the value the list of ecosystems (for OSV), or `""` / `null` (for NVD and GITHUB). And I think we're missing the GitHub event?

Reply:
Yes I deleted my change somehow while merging from main..